src/script/printer/ir_docsifier.h

```diff
@@ -114,7 +120,7 @@ class IRDocsifier : public ObjectRef {
  public:
   IRDocsifier(Map<String, String> ir_prefix);
 
-  using FType = ObjectFunctor<printer::Doc(const ObjectRef&, IRDocsifier)>;
+  using FType = ObjectFunctor<printer::Doc(const ObjectRef&, const ObjectPath&, IRDocsifier)>;
```

Comment:
Not a big deal for now but in the final PR should we just provide a traced object to dispatch function, and make the corresponding change in ObjectFunctor to make it able to dipatch/register based on the type id of the inner class of TracedObject?

Reply:
We might need to still store `printer::Doc(const ObjectRef&, const ObjectPath&, IRDocsifier)` just to be FFI-compatible, and expose a overloaded register function with `TracedObject<T>, IRDocsifier`.